chart/wordpress-site/templates/secret.yaml

```
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "wordpress-site.fullname" . }}
+  labels:
+    app.kubernetes.io/name: wordpress
+    app.kubernetes.io/part-of: wordpress
+    helm.sh/chart: {{ include "wordpress-site.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+type: Opaque
+data:
+  {{- if and .Values.code .Values.code.git }}
+  id_rsa: {{ required "A valid code.git.ssh_private_key entry required!" .Values.code.git.ssh_private_key | b64enc | quote}}
```

```suggestion
  id_rsa: {{ .Values.code.git.ssh_private_key | b64enc | quote}}
```

let it be optional. To use it with public repos